recipes/libnghttp2/all/conanfile.py

```diff
@@ -164,7 +161,7 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.components["nghttp2"].set_property("pkg_config_name", "libnghttp2")
-        suffix = "_static" if Version(self.version) > "1.39.2" and not self.options.shared else ""
+        suffix = "_static" if Version(self.version) == "1.40.0" and not self.options.shared else ""
```

Comment:
I even advice to add a patch for 1.40.0 in order to remove this `_static` suffix

Reply:
Force-pushed the changes

Reply:
I wonder why the linter is failing now. It seems that my change in `conandata.yml` is aligned with other lines in that file. Is that an outdated check?

Reply:
yes, it should be relaxed soon: https://github.com/conan-io/conan-center-index/issues/14865